src/samples/aspnet/Elsa.Samples.AspNet.WorkflowContexts/Program.cs

```diff
@@ -37,13 +37,18 @@
                 .UsePersistentVariables()
                 .UseBookmarkPersistence()
                 .UseWorkflowExecutionLogPersistence()
```

Comment:
These two lines are repeated. Is this intentional?

Reply:
Repeated?